submissions/VBystrov/html-movie-seat-booking/css/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Noto Sans', sans-serif;
+  font-size: 16px;
+}
+
+.wrapper {
+  display: flex;
+  justify-content: center;
+  height: 100vh;
```

Comment:
Fixed height not good practice in the blocks with various content. Use `min-height` instead. 

![image](https://user-images.githubusercontent.com/16196199/107156585-d29df200-6987-11eb-8791-7e53cff811ca.png)

Reply:
Done.